server/store/store_tests.go

```diff
@@ -510,10 +510,10 @@ func (pt *peerStoreTester) TestPeerStore(t *testing.T, cfg *DriverConfig) {
 	require.Equal(t, 7, s.NumSeeders(hash))
 	require.Equal(t, 3, s.NumLeechers(hash))
 
-	peers1, peers61, err := s.AnnouncePeers(hash, true, 5, peer, chihaya.Peer{})
+	_, _, err = s.AnnouncePeers(hash, true, 5, peer, chihaya.Peer{})
 	require.Nil(t, err)
-	require.NotNil(t, peers1)
-	require.NotNil(t, peers61)
+	// We can not test the returned slices. They change depending on the
```

Comment:
Rephrase this and put it above the `require.Nil`. Comments in Go always go above what they're describing.